bundles/admin/admin-announcements/Flyout.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { AnnouncementsList, AnnouncementsListHandler } from './view/';
+import { LocaleProvider } from 'oskari-ui/util';
+
+/**
+ * @class Oskari.admin.bundle.admin-announcements.Flyout
+ *
+ * Renders the admin-announcements flyout.
+ */
+Oskari.clazz.define('Oskari.admin.bundle.admin-announcements.Flyout',
+
+    /**
+     * @method create called automatically on construction
+     * @static
+     * @param {Oskari.admin.bundle.admin-announcements.Flyout} instance
+     *    reference to component that created the flyout
+     */
+    function (instance) {
+        this.instance = instance;
+        this.container = null;
+        this.AnnouncementsListHandler = new AnnouncementsListHandler(this.instance);
```

Comment:
The AnnouncementsListHandler contructor doesn't do anything with the parameter. Remove the instance parameter here to reduce complexity.
```suggestion
        this.AnnouncementsListHandler = new AnnouncementsListHandler();
```